static/app/components/events/interfaces/spans/traceView.tsx

```diff
@@ -21,24 +22,24 @@ type Props = {
   isEmbedded?: boolean;
 };
 
-class TraceView extends PureComponent<Props> {
-  traceViewRef = createRef<HTMLDivElement>();
-  virtualScrollBarContainerRef = createRef<HTMLDivElement>();
-  minimapInteractiveRef = createRef<HTMLDivElement>();
+function TraceView(props: Props) {
```

Comment:
curious if we should keep this as a `PureComponent` to avoid performance hits

Reply:
or i guess it's no longer pure?

Reply:
Oops good catch, I just converted this to a functional component because I needed to use a hook in it at one point, which has since been reverted. 

Now that this component has already been converted to an FC, would memoizing it be good enough to prevent performance hits?

Reply:
yup, that should do 👍